.github/workflows/robot/internal/bot/bot.go

```diff
@@ -145,3 +145,11 @@ func isAutoGeneratedFile(name string) bool {
 		strings.HasSuffix(name, "_pb.d.ts") ||
 		strings.Contains(name, "webassets/")
 }
+
+func isReleaseBranch(branch string) bool {
+	return strings.HasPrefix(branch, "branch/")
+}
+
+func isBotBackportBranch(branch string) bool {
+	return strings.HasPrefix(branch, "bot/backport")
```

Comment:
This is not a label, it's the name of the branch the bot creates when opening a backport.

Since we have to disable the "automatically delete branches when a PR is merged" we have a lot of remote branches building up. This should fix it.